spec/views/defendants/_offences.html.haml_spec.rb

```diff
@@ -13,19 +13,25 @@
       before { allow(offence).to receive(:mode_of_trial).and_return('Indictable only') }
 
       context 'with reason' do
+        let(:mot_reason_ostruct_collection) { mot_reasons_array.map { |el| OpenStruct.new(el) } }
+
+        let(:mot_reasons_array) do
+          [{ code: '4', description: 'Defendant elects trial by jury' }]
+        end
+
         before do
-          allow(offence).to receive(:mode_of_trial_reason).and_return('Court directs trial by jury')
+          allow(offence).to receive(:mode_of_trial_reasons).and_return(mot_reason_ostruct_collection)
         end
 
         it 'displays mode of trial with reason' do
           render
           expect(rendered).to have_css('.govuk-table__cell',
-                                       text: /Indictable only:\n*Court directs trial by jury/)
+                                       text: /Indictable only:\n*Defendant elects trial by jury/)
```

Comment:
If there is to be a line break between mode of trial and reason we should test fo this here and in the feature spec above

Reply:
👍 see comment above

Reply:
Here you are testing this exists
```
        %td.govuk-table__cell
          = "#{offence.mode_of_trial}:"
          %br
          = offence.mode_of_trial_reason_list
```
This is testing for 0 or more newline characters will not be testing anything. 

Does this pass the test?
```suggestion
                                       text: /Indictable only:<br>Defendant elects trial by jury/)
```

Reply:
@jsugarman it does not pass the test with a <br> . It looks like capybara/the call to render is replacing this with a newline? 

`expected to find visible css ".govuk-table__cell" with text /Indictable only:<br>Defendant elects trial by jury/ but there were no matches. Also found "\nNot available\n", "\nNot available\n", "\nIndictable only:\n\nDefendant elects trial by jury\n", which matched the selector but not all filters.`

Reply:
@kmahern have it test for 1+ newlines then (`\n+`).